netlify.toml

```diff
@@ -23,7 +23,11 @@
   from = "/discord"
   to = "http://discord.gg/GQbXUVCneJ"
   status = 301
+[[redirects]]
+  from = "/old"
+  to = "https://p4nth3rblog.netlify.app/"
+  status = 301
 [[plugins]]
 package = "netlify-plugin-inline-functions-env"
    [plugins.inputs]
-    include = ["GOOGLE_PRIVATE_KEY"]
```

Comment:
The removal and addition of this is stressing me out 😂

Reply:
That’s so bizzare 😂 I don’t remember deleting that line🤷🏻‍♂️